upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -1027,6 +959,86 @@ func (c *ApplyClusterCmd) validateKubernetesVersion() error {
 	return nil
 }
 
+// addFileAssets adds the file assets within the assetBuilder
+func (c *ApplyClusterCmd) addFileAssets(assetBuilder *assets.AssetBuilder) error {
```

Comment:
This refactor uses URLs instead of strings, adds file assets to the asset builder, and also reduced repeated code with a loop.